k2/python/csrc/torch/v2/ragged_arc.cu

```diff
@@ -250,6 +425,15 @@ bool RaggedArc::HasAttr(const std::string &name) const {
   return all_attr_names.count(name) > 0;
 }
 
+void RaggedArc::SetFiller(const std::string &name, float filler) {
+  fillers[name] = filler;
+}
+
+float RaggedArc::GetFiller(const std::string &name) const {
+  if (fillers.count(name) > 0) return fillers.at(name);
```

Comment:
I would recommend you to use `fillers.find()` since it needs only to lookup the key once.